src/main/scala/strawman/collection/immutable/TreeSet.scala

```diff
@@ -32,14 +42,11 @@ final class TreeSet[A]()(implicit val ordering: Ordering[A])
 
   // From SortedLike
   def range(from: A, until: A): TreeSet[A] = ???
-
-  // From SortedPolyTransforms
-  def map[B](f: (A) => B)(implicit ordering: Ordering[B]): TreeSet[B] = ???
```

Comment:
Trying to compile master without this line:

```
[info] Compiling 37 Scala sources to /Users/szeiger/code/collection-strawman/target/scala-2.12/classes...
[error] /Users/szeiger/code/collection-strawman/src/main/scala/strawman/collection/immutable/TreeSet.scala:10: class TreeSet needs to be abstract, since method map in trait SortedPolyTransforms of type [B](f: A => B)(implicit ordering: Ordering[B])strawman.collection.immutable.TreeSet[B] is not defined
[error] (Note that A => B does not match A => B: their type parameters differ)
[error] final class TreeSet[A]()(implicit val ordering: Ordering[A])
[error]             ^
[error] one error found
```